src/views/TemplateCustomPickerElement.vue

```diff
@@ -52,6 +50,10 @@
 				</NcButton>
 			</a>
 		</div>
+		<AddTemplateModal
+			ref="new-template"
+			:show.sync="showAddingModal"
+			@callback="getTemplates" />
```

Comment:
This `callback` event could be named `template-added` or something more explicit.